src/main/groovy/org/burnmydread/SimpleRandom.groovy

```diff
@@ -74,6 +98,13 @@ class SimpleRandom {
     }
 
     short unsignedNextShort() {
+        /* MP COMMENT
+         * This implementation only uses half the range of a unsigned short. 
+         * That may be fine as long as you document it.  Otherwise, you'll need
+         * grab an unsignedInt from the byte stream and then modulos it by Short.MAX_VALUE.
+         * Similar comment applies to unsignedInt.  For unsignedLong, you're out of luck
+         * in Java unless you want to use BigInt (probably not worth it).
+         */
```

Comment:
This and the latter comments are addressed by making the API match what I coded, so now the 'signed' calls are called nextPositiveOrNegitive and the 'unsigned' are called nextPositive